blacklist.go

```diff
@@ -63,7 +63,11 @@ func (b *Blacklist) Update() {
 	//get the lists to remove from the db
 	metasToRemove := getListsToRemove(b.lists, remoteMetas)
 	for _, metaToRemove := range metasToRemove {
-		b.db.RemoveList(metaToRemove)
+		err = b.db.RemoveList(metaToRemove)
```

Comment:
linter caught an unchecked error. Added handling for it similar to other places in the file.